OPHD/States/MapViewStateTurn.cpp

```diff
@@ -185,14 +185,11 @@ static RouteList findRoutes(micropather::MicroPather* solver, TileMap* tilemap,
 }
 
 
-static bool compareRoute(Route& a, Route& b) { return a.cost < b.cost; }
-
-
 static Route findLowestCostRoute(RouteList& routeList)
 {
 	if (routeList.empty()) { return Route(); }
 
-	std::sort(routeList.begin(), routeList.end(), compareRoute);
+	std::sort(routeList.begin(), routeList.end(), [](const Route& a, const Route& b) { return a.cost < b.cost; } );
```

Comment:
You could potentially find the lowest cost route without sorting. Sorting would have minimum O(n lg(n)) and more likely O(n^2) worst case efficiency, while searching should only require O(n). Assuming you only need the minimum cost, and don't care about the rest, that would improve efficiency.

Reply:
Sure, but this seems like mostly a moot point as it happens during turn calculation.

That said, finding the lowest cost route would still require looking at each route and keeping a running counter. Routes are usually only going to be a handful at a time so it seems like negligible cost. We can refactor as needed if bottle necks are identified during profiling.